### PR TITLE
adding example for creating PR from fork

### PR DESCRIPTION
**Pull request by shahabhijeet**
Sample for #1943

### CODE REVIEW

File: docs/demos/exploring-pull-requests.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
+### Creating new pull request from a fork
+```csharp
+public void CreatePR()
+{
+    /*
+    Scenario: Creating a pull request from a fork/branch (head) and merging into octokit.net/master (base)
```
Comment (shiftkey) [inline docs/demos/exploring-pull-requests.md]: Move this code comment out into the Markdown text

File: docs/demos/exploring-pull-requests.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+    When you are opening a PR against a repository within an organization, owner is the name of the organization
+    In this scenario as we are merging to octokit/octokit.net:master, owner=octokit
+    */
+    
+    GitHubClient ghClient = new GitHubClient(new ProductHeaderValue("MyLib", "v2.0.0"));
+    ghClient.Credentials = new Credentials("apiToken");
+
+    NewPullRequest newPr = new NewPullRequest("PrTitle", "forkName:branchName", "master");
+    Repository octokitRepo = ghClient.Repository.Get("octokit", "ocktokit.net").GetAwaiter().GetResult();
```
Comment (shiftkey) [inline docs/demos/exploring-pull-requests.md]:
```suggestion
    var octokitRepo = await ghClient.Repository.Get("octokit", "ocktokit.net");
```

File: docs/demos/exploring-pull-requests.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
+### Creating new pull request from a fork
+```csharp
+public void CreatePR()
```
Comment (shiftkey) [inline docs/demos/exploring-pull-requests.md]:
```suggestion
public async Task CreatePullRequestFromFork()
```

File: docs/demos/exploring-pull-requests.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+    In this scenario as we are merging to octokit/octokit.net:master, owner=octokit
+    */
+    
+    GitHubClient ghClient = new GitHubClient(new ProductHeaderValue("MyLib", "v2.0.0"));
+    ghClient.Credentials = new Credentials("apiToken");
+
+    NewPullRequest newPr = new NewPullRequest("PrTitle", "forkName:branchName", "master");
+    Repository octokitRepo = ghClient.Repository.Get("octokit", "ocktokit.net").GetAwaiter().GetResult();
+
+    PullRequest pr = ghClient.PullRequest.Create("octokit", "octokit.net", newPr).GetAwaiter().GetResult();
```
Comment (shiftkey) [inline docs/demos/exploring-pull-requests.md]:
```suggestion
    var pullRequest = await ghClient.PullRequest.Create("octokit", "octokit.net", newPr);
```

File: docs/demos/exploring-pull-requests.md [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+    //Or
+    PullRequest alternatePr = ghClient.PullRequest.Create(octokitRepo.Id, newPr).GetAwaiter().GetResult();
```
Comment (shiftkey) [inline docs/demos/exploring-pull-requests.md]:
```suggestion
    var alternatePr = await ghClient.PullRequest.Create(octokitRepo.Id, newPr);
```

**Comment by ryangribble** [comment]
Thanks @shahabhijeet

**Comment by ryangribble** [comment]
release_notes: Added sample covering how to create a PR from a fork

**Pull request merged**